### PR TITLE
[FMD-278]: consolidate funding data

### PR DESCRIPTION
**Pull request by Joel-Pearce**
Consolidates funding data as per: https://dluhcdigital.atlassian.net/browse/FMD-278

- [x] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### CODE REVIEW

**Comment by Joel-Pearce** [comment]
deployment failed

notes:

> 
the issue is that for some reason data_blob is being set as null on some rows when all that should be happening is some values in some keys should be being moved around, the syntax is wrong but it's quite a tricky one because it's JSONB manipulation with raw SQL, maybe just be easier to use ORM IMO

**Comment by albertkol** [comment]
Will the pipeline pass now? :popcorn:

File: db/migrations/versions/038_correct_fund_data.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
+"""empty message
```
Comment (samuelhwilliams) [inline db/migrations/versions/038_correct_fund_data.py:1]: 😢

**Review by gidsg — APPROVED** [review]
LGTM, we'll want to make sure these changes are communicated to the Fund Teams/end users 👍

File: db/migrations/versions/038_correct_fund_data.py (hunk 1 above)
Comment (albertkol) [reply db/migrations/versions/038_correct_fund_data.py]: I changed the message! I will not put up with this emoji passive aggressive feedback!

**Pull request merged**